docs/ui/components/Sidebar/SidebarLink.tsx

```diff
@@ -116,6 +116,14 @@ export const SidebarLink = ({ info, className, children }: SidebarLinkProps) =>
           BETA
         </div>
       )}
+      {info.isPreview && (
+        <div
+          className={mergeClasses(
+            '-mt-px ml-2 inline-flex h-[17px] items-center rounded-full bg-element px-[10px] text-[10px] font-normal leading-none text-secondary'
```

Comment:
Can we use the same color as "isBeta" tag? I don't think a gray background is eye-catching enough.

Reply:
Sure!